Src/WitsmlExplorer.Api/Jobs/DeleteTrajectoryStationsJob.cs

```diff
@@ -0,0 +1,11 @@
+using WitsmlExplorer.Api.Jobs.Common;
+using System.Collections.Generic;
+
+namespace WitsmlExplorer.Api.Jobs
+{
+    public record DeleteTrajectoryStationsJob
+    {
+        public TrajectoryReference Trajectory { get; init; }
```

Comment:
Use "TrajectoryStationReferences" here

Reply:
Remember to also update the job accordingly in frontend

Reply:
do you mean:
public TrajectoryStationReferences Trajectory { get; init; }
or
public TrajectoryStationReferences TrajectoryStation { get; init; }
?